tests/Functional/Adapter/TolerantParser/Util/NodeHelperTest.php

```diff
@@ -14,7 +14,7 @@ class NodeHelperTest extends TestCase
      */
     private $parser;
 
-    protected function setUp(): void
+    protected function setUp(): void: void
```

Comment:
There is a little bit of a mistake in the automation :)

Reply:
yes, it was a late night experiment :)